py_mind_memo/dialogs.py

```
@@ -0,0 +1,82 @@
+        self.destroy()
+
+    def show(self):
+        # 画面中央に配置
+        self.update_idletasks()
+        try:
+            parent = self.master
+            x = parent.winfo_x() + (parent.winfo_width() - self.winfo_reqwidth()) // 2
+            y = parent.winfo_y() + (parent.winfo_height() - self.winfo_reqheight()) // 2
+            self.geometry(f"+{x}+{y}")
+        except Exception:
+            pass
```

Ruff (0.15.7)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)